docs/developer.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Developer's guide
+=================
+
+About
+=====
+The Data Retriever Dashboard is divided into two parts:
+
+1. The Script
+2. The Dashboard
+
+**The Script**
+The script(dashboard_script.py in retrieverdash/retrieverdash/dashboard_script/ directory) is run on every Sunday at 12:00 AM using cron job. It generates a
+dataset_details.json file containing the details of the datasets and the diff files(html files showing
+side by side difference in the tables of a dataset if a change has been made to dataset).
+
+**The Dashboard**
+The dashboard provides an interface where maintainers and users can see the details of datasets(whether it is
+installing successfully or not using retriever and the diffs).
+
+Setting up locally
+==================
+
+Before starting to make contributions to Data Retriever Dashboard you first have to set it up
+locally on your machine. You’ll need Python 3.4+.
+
+**Steps to run Data Retriever Dashboard locally:**
+
+1. Clone the repository.
+2. From the directory containing manage.py, run the following command:
+   ``pip install -r requirements.txt`` to install the requirements for the dashboard.
+3. ``python manage.py makemigrations`` to make migrations.
+4. ``python manage.py migrate`` to migrate.
+5. ``python manage.py crontab add`` to add the cron job for running the script
```

Comment:
Could we be more specific on what we have to run when and why. We have the Helper script and the dashboard. 
So what should we run first.

I feel like steps 1-2 are fine but before we talk about step 3, we should a heading like
**To set up the dashboard and attach a cron job.**
3.
4.
5.

**To setup the dashboard and start running the job regardless of the set cron configuration.**
6....


change 7 to something like.
7. Open a browser and load the url that was provided when you run `python manage.py runserver`.

 (Note: When we are talking about `python manage.py runserver` in the point before 7, we should also mention that the url is default and if we want to change the port we could bind a new port number)